backend/onedrive/onedrive.go

```diff
@@ -605,7 +665,7 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 		opt:       *opt,
 		driveID:   opt.DriveID,
 		driveType: opt.DriveType,
-		srv:       rest.NewClient(oAuthClient).SetRoot(graphURL + "/drives/" + opt.DriveID),
+		srv:       rest.NewClient(oAuthClient).SetRoot(baseURLMap[graphURLID] + "/drives/" + opt.DriveID),
```

Comment:
You can then refer to the urls as `opt.AuthURL` and `opt.GraphURL` from the options.